esp/esp/program/modules/handlers/adminclass.py

```diff
@@ -396,6 +399,10 @@ def coteachers(self, request, tl, one, two, module, extra, prog):
             if cls.conflicts(teacher):
                 conflictinguser = teacher
             else:
+                if not RegistrationProfile.getLastForProgram(teacher, prog).teacher_info:
+                    lastProf = RegistrationProfile.getLastForProgram(teacher, prog)
+                    lastProf.teacher_info = TeacherInfo.addOrUpdate(teacher, lastProf, {'graduation_year': ''})
```

Comment:
Is there a reason you set graduation_year to `''`? Shouldn't we be passing all of the information that the teacher info should have? That includes `graduation_year`, `affiliation`, `major`, `shirt_size`, and `shirt_type`.